draft-ietf-httpbis-digest-headers.md

```diff
@@ -586,16 +586,19 @@ Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 
 ## Server Returns No Representation Data
 
+In this example, a HEAD request is used to retrieve the checksum
+of a resource.
+
 Requests without payload data can still send a `Digest` field
 applying the digest-algorithm to an empty representation.
+This can be useful in conjunction with signatures
+to ensure that the sender did not convey any payload
+(see {{usage-in-signatures}}).
 
 The response `Digest` field-value is calculated over the JSON object
```

Comment:
Per Julian's comment, we can also state "the request `Digest` field-value is calculated over ..."